urmzd/md-nlp
# NLP 03 - Create virtual enviroment

# Purpose
Closes #4 

# Changes
- Added documentation to explain usage of virtual environment.
- Create virtual environment files to lock Python version.

# Verification
- Execute `source venv/bin/activate` followed by `python --version` to check if `python` version is `3.9.5`
- Execute `deactivate` and `python --version` to ensure `python` version matches system version.
- Execute `pip freeze > requirements.txt` to ensure dependencies are seen in `requirements.txt`.
- Execute `pip install -r requirements.txt` to ensure dependencies are installed.
